manticore/ethereum/__init__.py

```diff
@@ -1,1657 +1,16 @@
-import random
-import binascii
-import string
-import re
-import os
-import pyevmasm as EVMAsm
+from manticore.utils.install_helper import ensure_evm_deps
```

Comment:
Most or all of this file was moved to `ethereum/manticore.py`.
The `FilterFunction` was moved to `ethereum/plugins.py`